internal/api/api.go

```diff
@@ -155,5 +163,5 @@ func (api *idler) getURLAndToken(r *http.Request) (string, string, error) {
 	if ok {
 		return openShiftAPIURL, bearerToken, nil
 	}
-	return "", "", nil
+	return "", "", fmt.Errorf("Unknown or invalid OpenShift API URL")
```

Comment:
Shouldn't we return nil instead of empty strings? It's not critical, It doesn't break anything, you can skip it as well. I know throughout the project there are many functions like this. We can probably change them as we face them.
WDYT?

Reply:
heu nil is not a str type compatible,

Reply:
but you can make it a pointer

Reply:
I am not sure what's the point, this is very much nitpicking,